packages/modal/src/presenters/ModalPresenter.test.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { generateId } from "@hig/utils";
+import ModalPresenter from "./ModalPresenter";
+
+describe("checkbox/presenters/ModalPresenter", () => {
+  afterEach(() => {
+    generateId.mockReset();
+  });
+
+  [
+    {
+      description: "renders without props",
+      props: {}
+    },
+    {
+      description: "renders with header children",
+      props: {
+        children: [<p key={"p"}>Body</p>],
```

Comment:
_nitpick:_ No need to escape strings in JSX. I'm surprised our linterisn't catching this.

```jsx
<p key="p">Body</p>
```

Reply:
Updated in 5e9a1643